<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QWidget" name="Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="welcomeLabel">
      <property name="text">
       <string>Welcome, User!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="incomeEntryButton">
      <property name="text">
       <string>Income Entry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="expenseEntryButton">
      <property name="text">
       <string>Expense Entry</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="totalBalanceButton">
      <property name="text">
       <string>Total Balance</string>
      </property>
     </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
